src/main/java/com/znsio/rpi/utils/ScreenShotManager.java

```diff
@@ -0,0 +1,101 @@
+package com.znsio.rpi.utils;
+
+import org.apache.commons.lang3.exception.ExceptionUtils;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+
+public class ScreenShotManager {
+    private static final String directoryPath = System.getProperty("user.dir") + File.separator +
+            "TestReport/Screenshots";
+    private static final Logger LOGGER = Logger.getLogger(ScreenShotManager.class.getSimpleName());
+
+    public static String captureScreenShot(WebDriver webDriver) {
```

Comment:
Where/how are these methods used?

Reply:
They are used in getting-started-with-rp-ap repo for capturing and logging screenshots to ReportPortal. Have given details on this in Readme.md file as well.

Reply:
Could we not do this as a method in ReportPortalLogger - ex: captureAndAttachScreenshot(....) ?

Reply:
You're asking to move the entire implementation of ScreenShotManager class to ReportportalLogger? There are quite a few utilities we written here and I've grouped them all here as it comes under the umbrella of processing of Screenshot. Let me know your thoughts?

Reply:
Implemented the captureAndAttachScreenshot method in ReportPortalLogger so that consumer doesn't have to import ScreenShotManager for logging screenshots to ReportPortal